src/app/pages/organizations/developers/developer/join.jsx

```diff
@@ -22,10 +24,47 @@ import { useFetchDevelopers, usePutJoinDevelopers } from 'api/developer';
 import { ChplActionBar } from 'components/action-bar';
 import { ChplTextField } from 'components/util';
 import { getAngularService } from 'services/angular-react-helper';
+import ClearIcon from '@material-ui/icons/Clear';
+import AddIcon from '@material-ui/icons/Add';
+import { theme } from 'themes';
 
 const useStyles = makeStyles({
   headingPadding: {
     padding: '16px 0',
+    backgroundColor: '#fff',
+  },
+  boxContainer: {
```

Comment:
I think classes should be named for their purpose, not for their technical impact. I mention it because I think these "Box" elements could be replaced with DIVs, and if we did that the name would have to change. If, instead, the class was something like "pageContainer", that would be a meaningful name that wouldn't have to be updated if the implementation changed